workflow/react-project/src/components/common/Modal/ModalContent/RandomCocktail/index.jsx

```diff
@@ -0,0 +1,56 @@
+import { useSelector } from 'react-redux';
+
+import { faStar } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import './style.less';
+
+const RandomCocktail = () => {
+  const randomCocktailStatus = useSelector(state => state.randomCocktail?.status);
+  const randomCocktailDetails = useSelector(state => state.randomCocktail?.details)?.drinks?.[0];
```

Comment:
let's use Selectors for getting data from store.
[Here you can learn about Selectors](https://redux-toolkit.js.org/api/createSelector) (we will use `Reselect` library)

As rule we create separate file for selectors in `store` folder.

So you will be able do like this: `const loading = useSelector(loadingSelector);`